plugins/mac-videotoolbox/encoder.c

```diff
@@ -939,12 +1129,9 @@ static bool rate_control_limit_bitrate_modified(obs_properties_t *ppts,
 	return true;
 }
 
-static obs_properties_t *vt_properties(void *unused, void *data)
+static void vt_properties_bitrate(obs_properties_t *props,
```

Comment:
Yeah, I guess this was extracted before because it was used by multiple functions, but with H.264 and HEVC being handled by the same function, it can be inlined.